common_data_for_test.go

```diff
@@ -159,6 +159,16 @@ var tests = []TestData{
 			BinaryFormat:   []byte{98, 112, 108, 105, 115, 116, 48, 48, 85, 72, 101, 108, 108, 111, 8, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 14},
 		},
 	},
+	{
+		Name:  "String containing aposthrope",
+		Value: "'",
```

Comment:
I think an example like `JSON's` would make this test case more straightforward.